eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -501,61 +516,62 @@ jobs:
         displayName: Prep the Build
         workingDirectory: $(sourcesPath)
 
-    - script: |
-        set -ex
-        df -h
+    - ${{ if ne(parameters.skipBuild, 'true') }}:
```

Comment:
I think this another refcount for componentizing parts of this YML, i.e. to have a step just for running tests. @NikolaMilosavljevic do we have a tracking issue for that?

Reply:
https://github.com/dotnet/source-build/issues/5037